Packages/Core/Runtime/ValueLists/BaseAtomValueList.cs

```diff
@@ -14,6 +14,11 @@ public abstract class BaseAtomValueList : BaseAtom
         public AtomEventBase Cleared;
         protected abstract IList IList { get; }
 
+        /// <summary>
+        /// Whether the list should start cleared
+        /// </summary>
+        public bool StartCleared;
```

Comment:
You probably don't want other classes to fiddle with this Bool. Should really be `private` with `[SerializeField]`